src/siphash.c

```diff
@@ -36,7 +36,7 @@
    ((uint64_t)((p)[4]) << 32) | ((uint64_t)((p)[5]) << 40) |                   \
    ((uint64_t)((p)[6]) << 48) | ((uint64_t)((p)[7]) << 56))
 
-uint64_t siphash(char key[16], char data[], int c, int d){
+uint64_t siphash(const char key[16], char data[], unsigned int c, unsigned int d, size_t data_len){
```

Comment:
No `const` for `char data[]`?

Reply:
Good catch! That one must've slipped by. I'm gonna fix it rn.